modules/cli/src/main/java/org/apache/ignite/cli/deprecated/spec/NodeCommandSpec.java

```diff
@@ -60,45 +63,22 @@ public class NodeCommandSpec {
     /**
      * Starts Ignite node command.
      */
-    @Command(name = "start", description = "Starts an Ignite node locally.")
+    @Command(name = "start", description = "Starts an Ignite node locally")
     @Singleton
     public static class StartNodeCommandSpec extends BaseCommand implements Callable<Integer> {
 
+        /** Consistent id, which will be used by new node. */
+        @Parameters(paramLabel = "name", description = "Name of the new node")
+        public String nodeName;
         /** Loader for Ignite distributive paths. */
         @Inject
         private CliPathsConfigLoader cliPathsCfgLdr;
-
```

Comment:
Looks like some of the line breaks got removed during the merge and ConfigOptions class is again at the bottom.